developers/ide/vscode.md

```diff
@@ -5,7 +5,87 @@ title: Visual Studio Code
 
 # VSCode
 
-::: tip TODO
-We are currently reworking how to setup a perfect development enviroment.
-A new step by step guide will appear here soon.
-:::
+The following shows the steps for a Win10 system but should be similar for OSX/Linux.  Please note that you must double up on backward slashes on a windows system (`c:\\something`).  On linux/osx, simply use a forward slash (`\etc\something`) .
+
+## Pre-requesites
+
+1. Must have Visual Studio Code (VSCode) installed
+2. Java 8+ SDK (not the JRE) installed and either JAVA_HOME or JDK_HOME pointing to it
+3. Base development directory (example: `c:\users\me\development`).  The `%BASE%` references below will be to this directory.
+4. Maven installed and the `mvn` command is in your path (open a console and type `mvn -version` to verify)
+5. openHAB 2.5+ is already installed
```

Comment:
Can you move all pre-requesites except VSCode from here to the overview section please. Those are true for all IDEs.